crates/forge-runner/src/forge_config.rs

```diff
@@ -0,0 +1,68 @@
+use std::num::NonZeroU32;
+use std::sync::Arc;
+
+#[derive(Debug, PartialEq)]
+pub struct ForgeConfig {
+    pub runner_config: Arc<RunnerConfig>,
+    pub runtime_config: Arc<RuntimeConfig>,
+    pub output_config: OutputConfig,
+}
+
+#[derive(Debug, PartialEq)]
+pub struct RunnerConfig {
+    pub exit_first: bool,
+    pub fuzzer_runs: NonZeroU32,
+    pub fuzzer_seed: u64,
+}
```

Comment:
Do we need this to be separated?

Reply:
I agree with @Arcticae, the rationale of the separation is ambigous, we should make it more ordered.

I don't understand what `RunnerConfig`, `RuntimeConfig` and `ContextData` mean exactly.

Reply:
https://github.com/foundry-rs/starknet-foundry/pull/2032#discussion_r1569220151